### DISCUSSION TITLE
Welcome to formulaone-card Discussions!

### DISCUSSION THREAD

**Discussion by marcokreeft87** [marcokreeft87/formulaone-card#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by acnsx** [comment]
New to GetHub and a HA user.   This "out of the box" card really personalizes my dashboard.

Just a quick comment.   I LOVE IT!    Thanks so much.   I'm hooked and look forward to enhancements and will spread the word like a country preacher on Sunday.

**Reply by marcokreeft87** [reply]
Thnx haha!

**Comment by JeroenvanA** [comment]
I'm not supposed to see the new season right? It's still saying season is over from last year. Or do I have to change something?

**Reply by marcokreeft87** [reply]
No that's correct. Don't know if the new season will be provided by the api

**Reply by JeroenvanA** [reply]
Oh that's why. Thanks for the quick reply!

**Reply by acla6525** [reply]
Fingers crossed they do run the API! I was wondering the same thing.

**Reply by jspanitz** [reply]
Going to be a very sad day if they don't make 24 available - is there a contact for request as much or just general social media contacts?
